Added turnip tests

mikhaillopandia opened the pull request.

Commit:
Added turnip tests

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+Feature: The signin process
+  Scenario: I can open login page via menu
+    Given there is a home page
```
Given open home page
vs
Given I am on home page

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+  Scenario: I can open login page via menu
+    Given there is a home page
+    When I click login
+    Then login page should open
```
Then I should see login page

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+
+  Scenario: I can login with correct credentials
+    Given there is a login page
+    When I fill 'lopandya96@gmail.com' in email and 'lopandya96' in password
```
When I fill out correct email and password

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+  Scenario: I can login with correct credentials
+    Given there is a login page
+    When I fill 'lopandya96@gmail.com' in email and 'lopandya96' in password
+    And click enter
```
And I press enter key

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+    Given there is a login page
+    When I fill 'lopandya96@gmail.com' in email and 'lopandya96' in password
+    And click enter
+    Then login should be successful
```
Then I should be logged in successfully

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+Feature: The signin process
+  Scenario: I can open login page via menu
```
User can ...

romikoops commented on spec/acceptance/features/signin.feature:
```diff
@@ -0,0 +1,47 @@
+    When I click login
+    Then login page should open
+
+  Scenario: I can login with correct credentials
```
Scenario: User can ...

romikoops commented on spec/acceptance/steps/steps.rb:
```diff
@@ -0,0 +1,32 @@
+end
+
+step 'login should be successful' do
+  expect(page).to have_content 'Signed in successfully.'
```
expect(page).to have_content('Signed in successfully.')

romikoops commented on spec/signin_spec.rb:
```diff
@@ -81,7 +81,7 @@
     context "when incorrect data" do
       given(:incorrect_credentials) { {email: 'FALSE@i.ua', password: 'FALSE'} }
-      scenario "user can not login with incorrect password" do
+      scenario "user can not login with incorrect data" do
```
Remove old spec files

Commit:
Turnip corrections

According to Conversation in https://github.com/mikhaillopandia/autotest2/pull/1 changes has been applied.

romikoops commented on spec/acceptance/steps/steps.rb:
```diff
@@ -1,32 +1,54 @@
 end
 
-step 'I fill :email in email and :password in password' do |email, password|
+step "I fill out :email email and :password password" do |email, password|
```
Do not mix several steps into single one. It introduces extra complexity in step definitions which should be ideally 3 lines of code

Commit:
Steps changes

Reffactored steps into common and sign_in.
Step 'I fill out :type email and :type password' has been decomposed into 2.

romikoops commented on spec/acceptance/steps/sign_in_steps.rb:
```diff
@@ -0,0 +1,39 @@
+    expect(page).to have_current_path('/users/sign_in')
+  end
+
+  step "I fill out :type email" do |type|
```
```ruby
step 'I fill out correct email' do |type|
end

step 'I fill out wrong email' do |type|
end
```

romikoops commented on spec/acceptance/steps/sign_in_steps.rb:
```diff
@@ -0,0 +1,39 @@
+    fill_in('user[email]', :with => email)
+  end
+
+  step "I fill out :type password" do |type|
```
same

Commit:
Corrected email/password fill out steps

Commit:
Spaces corrections for rubocop
